storm-core/src/jvm/org/apache/storm/daemon/supervisor/SupervisorUtils.java

```diff
@@ -104,6 +104,16 @@ public static void setupStormCodeDir(Map<String, Object> conf, Map<String, Objec
         }
     }
 
+    public static void setupWorkerArtifactsDir(Map<String, Object> conf, Map<String, Object> stormConf, String dir) throws IOException {
+        if (Utils.getBoolean(conf.get(Config.SUPERVISOR_RUN_WORKER_AS_USER), false)) {
+            String logPrefix = "setup conf for " + dir;
```

Comment:
No idea. I copied this over from setupStormCodeDir so they would look similar in the logs.
Can change both to actually make sense.